packages/langchain/src/orchestration/__snapshots__/client.test.ts.snap

```diff
@@ -137,3 +137,177 @@ exports[`orchestration service client returns successful response when timeout i
+    },
+    "content": "The SAP Cloud SDK is a comprehensive development toolkit designed to simplify and accelerate the creation of applications that integrate with SAP solutions, particularly those built on the SAP Business Technology Platform (BTP). It provides developers with libraries, tools, and best practices that streamline the process of connecting to SAP systems, such as S/4HANA and other services available on the SAP Cloud Platform.
+
+Key features of the SAP Cloud SDK include:
+
+1. **Simplified Connectivity**: The SDK offers pre-built libraries to easily interact with SAP services, providing capabilities for authentication, service consumption, and OData/REST client generation.
+
+2. **Multi-cloud Support**: It supports multiple cloud environments, ensuring that applications remain flexible and can be deployed across various cloud providers.
+
+3. **Best Practices and Guidelines**: The SDK includes best practices for development, ensuring high-quality, scalable, and maintainable code.
+
+4. **Project Scaffolding and Code Samples**: Developers can quickly start their projects using provided templates and samples, accelerating the development process and reducing the learning curve.
+
+5. **Extensive Documentation and Community Support**: Ample documentation, tutorials, and an active community help developers overcome challenges and adopt the SDK efficiently.
+
+Overall, the SAP Cloud SDK is an essential tool for developers looking to build cloud-native applications and extensions that seamlessly integrate with SAP's enterprise solutions.",
+    "id": undefined,
+    "invalid_tool_calls": [],
+    "response_metadata": {
+      "completion": 0,
+      "finish_reason": "stop",
+      "prompt": 0,
+      "token_usage": {
+        "completion_tokens": 271,
+        "prompt_tokens": 17,
+        "total_tokens": 288,
+      },
+    },
+    "tool_call_chunks": [],
+    "tool_calls": [],
+    "usage_metadata": {
+      "input_tokens": 17,
+      "output_tokens": 271,
+      "total_tokens": 288,
+    },
+  },
+  "lc": 1,
+  "type": "constructor",
+}
+`;
+
+exports[`orchestration service client supports streaming responses with tool calls 1`] = `
+{
+  "id": [
+    "langchain_core",
+    "messages",
+    "AIMessageChunk",
+  ],
+  "kwargs": {
+    "additional_kwargs": {
+      "module_results": {
+        "llm": {
+          "choices": [
+            {
+              "delta": {
+                "content": "",
+                "role": "assistantassistantassistantassistantassistant",
```

Comment:
Some properties are concatenated multiple times, as `concat()` in `AIMessageChunk` does not have these properties. We need to custom handle these properties during concatenation. Will handle this in a follow-up BLI: https://github.com/SAP/ai-sdk-js-backlog/issues/322